src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.UserRegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String UNIQUE_LOGIN = "arsenvovk";
+    private static final String USED_LOGIN = "bobhomenko";
+    private static final String VALID_PASSWORD = "1vtqn9frf";
+    private static final String INVALID_PASSWORD = "qwer";
+    private static final Integer ADULT_AGE = 27;
+    private static final Integer YOUNG_AGE = 17;
+    private static final Integer INVALID_AGE = -8;
+    private static StorageDao storageDao;
+    private static RegistrationService registrationService;
+    private User testUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+    }
```

Comment:
You should use the same storage that `registrationService` uses, that's why you have static `Storage.people` list, use it directly
```suggestion
    static void beforeAll() {
        registrationService = new RegistrationServiceImpl();
    }
```